src/execution/operator/csv_scanner/parallel_csv_reader.cpp

```diff
@@ -452,6 +440,10 @@ add_row : {
 			goto final_state;
 		}
 		SkipEmptyLines();
+		if (position_buffer - verification_positions.end_of_last_line > options.buffer_size) {
+			error_message = "Line does not fit in one buffer. Increase the buffer size.";
```

Comment:
Okay, that makes sense. At that point, we assume that the user knows that they have massive lines in their CSV file :) Is the 32MB threshold documented transparently with the option to increase the buffer size?

Reply:
A quick Google search did not lead to anything for me, except this: https://github.com/duckdb/duckdb/discussions/8102

Reply:
@szarnyasg This is something we should indeed consider documenting, possibly after merging this PR. The current rule (in this PR) is that by default, buffers will have 32Mb, unless the file is smaller than 32Mb (In that case it will have the same size as the file size). The `buffer_size` option then allows users who know what they are doing to override this.